qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestData.java

```diff
@@ -684,4 +678,60 @@ public String toString() {
 		public String datetime;
 		public String uuid;
 	}
+
+	public static class PatientGenerator {
+
+		private static final String[] PATIENT_GIVEN_NAMES = {"Alexandre", "Achint",
+				"Brittany", "Burke", "Cesar", "Cosmin", "Daniel", "Darius",
+				"David", "Ellen", "Émerson", "Evan", "Fernando", "Gabou",
+				"Glauber", "Hamish", "Louise", "Mário", "Mark", "Natália", "Neil",
+				"Neissi", "Nelice", "Rafal", "Renee", "Wyclif"};
+		private static final String[] PATIENT_FAMILY_NAMES = {"Barbosa", "Sethi",
+				"Eddy", "Mamlin", "Vortmann", "Ioan", "Kayiwa", "Jazayeri",
+				"Walton", "Ball", "Hernandez", "Waters", "Freire", "Mandy",
+				"Ramos", "Fraser", "Sécordel", "Areias", "Goodrich", "Arsand",
+				"Craven", "Lima", "Heck", "Korytkowski", "Orser", "Luyima"};
+		private static final String[] PATIENT_BIRTH_MONTH = MONTHS;
+		private static final String[] PATIENT_BIRTH_DAY = DAYS;
+		private static final String[] PATIENT_BIRTH_YEAR = YEARS;
+		private static final String[] PATIENT_GENDER = {"Male", "Female"};
+
+		public static org.openmrs.contrib.qaframework.helper.TestPatient generateTestPatient() {
+			org.openmrs.contrib.qaframework.helper.TestPatient p = new org.openmrs.contrib.qaframework.helper.TestPatient();
```

Comment:
couldn't this be refactored to be rather an import?

Reply:
Since this class is not coming from any package. We could simply use it as if it comes from the very class. Have tested out this and all is fine. Another option would be refactoring the entire code which might bring in more code duplicates. cc @kdaud Let me know your suggestion on this. Thanks @Muta-Jonathan